gate-iap/src/main/java/com/netflix/spinnaker/gate/security/iap/IapSsoConfig.java

```diff
@@ -95,6 +95,8 @@ public FilterRegistrationBean iapFilterRegistration(IAPAuthenticationFilter filt
     String issuerId = "https://cloud.google.com/iap";
     String audience;
     String iapVerifyKeyUrl = "https://www.gstatic.com/iap/verify/public_key-jwk";
+    long issuedAtTimeAllowedSkew = 30000L;
```

Comment:
Might as well rename to IapSecurityConfigProperties to be consistent

Reply:
Nice catch, didn't see that one.